.github/workflows/pr-build-and-e2e-tests.yml

```diff
@@ -127,6 +131,8 @@ jobs:
 
             - name: Load docker images and start containers.
               working-directory: plugins/woocommerce
+              env:
+                ENABLE_HPOS: 1
```

Comment:
Think this is a mistake to enable only for performance in this regular PR workflow?

Reply:
That's right! Hanks for catching that